integreat_compass/core/circleci_settings.py

```diff
@@ -0,0 +1,14 @@
+"""
+Django settings for our CircleCI workflow.
+All configurations are imported from :mod:`~integreat_compass.core.settings`.
+For more information on this file, see :doc:`django:topics/settings`.
+For the full list of settings and their values, see :doc:`django:ref/settings`.
+"""
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
+from .settings import *
+
+#: Set a dummy secret key for CircleCI build even if it's not in debug mode
+SECRET_KEY = "dummy"
+#: Use debug logging on CircleCI
```

Comment:
Please remove this comment, it adds no info at all

Reply:
Is applied now :)